README.md

```diff
@@ -146,6 +146,7 @@ FakerElixir.Avatar.robohash("ahahahaha", "198x198", "jpg", "set2", "bg2") # "htt
 ### FakerElixir.Bank
```

Comment:
Well that's my bad, I did let ```@credit_card_vendors ["Visa", "MasterCard"]``` in the locales but it's not used. This method already exists and it's called ```FakerElixir.Bank.credit_card_type # "MasterCard```